### PR TITLE
feat: Add fr-FR translation and begin translation with dashboard and clients

### LINKED ISSUE #2: translate dashboard and clients container
- Translate elements from Dashboard (/)
- Translate elements from ClientsContainer (/clients)
- Translate size units (in commons)

### PR DESCRIPTION
**Pull request by FlorianRuen**
Closes #2

### CODE REVIEW

**Review by nicobao — CHANGES_REQUESTED** [review]
Nice!
You need to add `clients.json` too

File: cidgravity/locales/dev/clientsContainer.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    "newClient": "Create new client",
+    "modifyLimits": "Modify"
+  },
+  "globalLimits": {
+    "storage": {
+      "storageDeals": "Number of storage deals",
+      "unlimited": "unlimited",
+      "cumulativeStorageDeals": "Cumulative storage deal",
+      "deals_zero": "0 deal",
+      "deals_one": "1 deal",
```
Comment (nicobao) [inline cidgravity/locales/dev/clientsContainer.json:14]: Replace to:
```
"deals_zero": "{{count}} deal",
"deals_one": "{{could}} deal"
```

File: cidgravity/locales/dev/clientsContainer.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      "deals_two": "{{count}} deals",
+      "deals_few": "{{count}} deals",
+      "deals_many": "{{count}} deals",
+      "deals_other": "{{count}} deals"
+    },
+    "retrieval": {
+      "retrievalDeals": "Number of retrieval deals",
+      "unlimited": "unlimited",
+      "deals_zero": "0 deal",
+      "deals_one": "1 deal",
```
Comment (nicobao) [inline cidgravity/locales/dev/clientsContainer.json:24]:
```
"deals_zero": "{{count}} deal",
"deals_one": "{{could}} deal"
```

File: cidgravity/locales/dev/clientsContainer.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      "blockClient": "Block client",
+      "unblockClient": "Unblock client"
+    }
+  },
+  "errors": {
+    "unableToLoad": "Unable to load existing clients. Try again in few minutes!",
+    "noClientFound": "You haven't any client under your account.",
+    "addPricingModelBefore": "Before add a new client, be careful to create your pricing models"
+  },
+  "toasts": {
```
Comment (nicobao) [inline cidgravity/locales/dev/clientsContainer.json:51]: Julien wanted to rename `toasts` to `notifications`

File: cidgravity/locales/dev/common.json [hunk 4]
```diff
@@ -1,7 +1,33 @@
 {
   "common": {
-    "button": {
+    "buttons": {
       "confirm": "Confirm"
+    },
+    "sizeUnits": {
+      "B_zero": "0 B",
+      "B_one": "1 B",
```
Comment (nicobao) [inline cidgravity/locales/dev/common.json:8]: replace here too

File: cidgravity/locales/dev/common.json [hunk 5]
```diff
@@ -1,7 +1,33 @@
...
+      "B_two": "{{count}} B",
+      "B_few": "{{count}} B",
+      "B_many": "{{count}} B",
+      "B_other": "{{count}} B",
+      "GiB_zero": "0 GiB",
+      "GiB_one": "1 GiB",
```
Comment (nicobao) [inline cidgravity/locales/dev/common.json:14]: replace here too

File: cidgravity/locales/dev/common.json [hunk 6]
```diff
@@ -1,7 +1,33 @@
...
+      "GiB_two": "{{count}} GiB",
+      "GiB_few": "{{count}} GiB",
+      "GiB_many": "{{count}} GiB",
+      "GiB_other": "{{count}} GiB",
+      "TiB_zero": "0 TiB",
+      "TiB_one": "1 TiB",
```
Comment (nicobao) [inline cidgravity/locales/dev/common.json:20]: replace here too

File: cidgravity/locales/dev/common.json [hunk 7]
```diff
@@ -1,7 +1,33 @@
...
+      "TiB_two": "{{count}} TiB",
+      "TiB_few": "{{count}} TiB",
+      "TiB_many": "{{count}} TiB",
+      "TiB_other": "{{count}} TiB",
+      "MiB_zero": "0 MiB",
+      "MiB_one": "1 MiB",
```
Comment (nicobao) [inline cidgravity/locales/dev/common.json:26]: replace here too

**Review by nicobao — CHANGES_REQUESTED** [review]
Can you push `.prettierrc.json` and the update `package.json` and `package-lock.json`?

File: cidgravity/locales/dev/common.json (hunk 7 above)
Comment (FlorianRuen) [reply cidgravity/locales/dev/common.json:26]: Done

File: cidgravity/locales/dev/common.json (hunk 6 above)
Comment (FlorianRuen) [reply cidgravity/locales/dev/common.json:20]: Done

File: cidgravity/locales/dev/common.json (hunk 4 above)
Comment (FlorianRuen) [reply cidgravity/locales/dev/common.json:8]: Done

File: cidgravity/locales/dev/clientsContainer.json (hunk 3 above)
Comment (FlorianRuen) [reply cidgravity/locales/dev/clientsContainer.json:51]: Done

File: cidgravity/locales/dev/clientsContainer.json (hunk 2 above)
Comment (FlorianRuen) [reply cidgravity/locales/dev/clientsContainer.json:24]: Done

File: cidgravity/locales/dev/clientsContainer.json (hunk 1 above)
Comment (FlorianRuen) [reply cidgravity/locales/dev/clientsContainer.json:14]: Done

**Comment by FlorianRuen** [comment]
> Nice! You need to add `clients.json` too

Why clients.json ? I don't have a file with that name

**Comment by FlorianRuen** [comment]
> Can you push `.prettierrc.json` and the update `package.json` and `package-lock.json`?

Good point, I hadn't noticed that they hadn't been added to the commit

**Review by nicobao — APPROVED** [review]
LGTM

**Pull request merged**